integration-tests/http/jest.config.js

```diff
@@ -1,7 +1,6 @@
 process.chdir(__dirname)
 
 module.exports = {
-  name: "HTTP Integration Tests",
```

Comment:
Q: why dont we need that anymore? what was it used for?

Reply:
didn't see it being used anywhere and the new jest config doesn't support it